cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,50 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.5.5 <http://docs.cask.co/cdap/3.5.5/index.html>`__
+=============================================================
+
+New Features
+------------
+- :cask-issue:`CDAP-4535` - Authentication server announce address is now configurable
+  with the property ``security.auth.server.announce.urls``, which are comma-separated URLs in
+  the form of ``protocol://host:port``. The property
+  ``security.auth.server.announce.address`` is now deprecated. It is only used if it is set
+  but ``security.auth.server.announce.urls`` has not been set.
+  ``security.auth.server.announce.address`` takes a single address in the form of
```

Comment:
No, this has not been documented in this detail before, nor elsewhere in the documentation. There is also a change in that the deprecated property (``security.auth.server.announce.address``) now accepts either `host:port` or just `host`.